File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/Context.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Context {

    @SerializedName("location")
    @Expose
    private Location location;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("href")
    @Expose
    private String href;

    public Location getLocation() {
        return location;
    }

    public void setLocation(Location location) {
        this.location = location;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getHref() {
        return href;
    }

    public void setHref(String href) {
        this.href = href;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverExplore/Context.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore

import com.google.gson.annotations.Expose

class Context {
    @SerializedName("location")
    @Expose
    private var location: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Location? = null

    @SerializedName("type")
    @Expose
    private var type: String? = null

    @SerializedName("href")
    @Expose
    private var href: String? = null
    fun getLocation(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Location? {
        return location
    }

    fun setLocation(location: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Location?) {
        this.location = location
    }

    fun getType(): String? {
        return type
    }

    fun setType(type: String?) {
        this.type = type
    }

    fun getHref(): String? {
        return href
    }

    fun setHref(href: String?) {
        this.href = href
    }
}